money-core/src/main/scala/com/comcast/money/core/async/AsyncNotificationHandlerFactory.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.core.async
+
+import com.typesafe.config.Config
+
+object AsyncNotificationHandlerFactory {
+
+  def create(config: Config): AsyncNotificationHandler = {
```

Comment:
What about `object AsyncNotificationHandler` and `create` as `apply` would that look bad at call site? I guess this way you can just `import AsyncNotificationHandlerFactory.create`